docs/configuration-v3_0.md

```diff
@@ -1,4 +1,5 @@
 ---
+# This file is automatically generated; do not edit
```

Comment:
Maybe we could indicate what generates it so that we route them to the right place?